includes/class-wc-payments-token-service.php

```diff
@@ -44,26 +46,33 @@ public function __construct( WC_Payments_API_Client $payments_api_client, WC_Pay
 		add_action( 'woocommerce_payment_token_deleted', [ $this, 'woocommerce_payment_token_deleted' ], 10, 2 );
 		add_action( 'woocommerce_payment_token_set_default', [ $this, 'woocommerce_payment_token_set_default' ], 10, 2 );
 		add_filter( 'woocommerce_get_customer_payment_tokens', [ $this, 'woocommerce_get_customer_payment_tokens' ], 10, 3 );
+		add_filter( 'woocommerce_payment_methods_list_item', [ $this, 'get_account_saved_payment_methods_list_item_sepa' ], 10, 2 );
 	}
 
 	/**
 	 * Creates and add a token to an user, based on the payment_method object
 	 *
 	 * @param array   $payment_method Payment method to be added.
 	 * @param WP_User $user           User to attach payment method to.
-	 * @return WC_Payment_Token_CC    The WC object for the payment token.
+	 * @return WC_Payment_Token_CC|WC_Payment_Token_Sepa    The WC object for the payment token.
```

Comment:
Doc-block alignment.

Reply:
Yeah, the fixer is not spotting this sort of thing, unfortunately. The Idea is to rely on the fixer, but align descriptions vertically.